### PR TITLE
Add sns link and describe changefeed_creation_timestamp function

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-11479

This PR adds:

- a link to a blog on connecting CRDB changefeeds to Amazon SNS.
- further description to the `changefeed_creation_timestamp()` function.
- updates wording on the cdc overview page.

v24.2 + v24.3.

### CODE REVIEW

File: src/current/v24.2/cdc-queries.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following table outlines functions that are useful with CDC queries:
 
 Function                  | Description
 --------------------------+----------------------
-`changefeed_creation_timestamp()` | Returns the decimal MVCC timestamp when the changefeed was created.
+`changefeed_creation_timestamp()` | Returns the decimal MVCC timestamp when the changefeed was created. Use this function to build CDC queries that restrict emitted events by time. `changefeed_creation_timestamp()` can serve a similar purpose to the [`now()` time function]({% link {{ page.version.version }}/functions-and-operators.md %}#date-and-time-functions), which is not supported with CDC queries.
```
Comment (rohan-joshi) [inline src/current/v24.2/cdc-queries.md:64]: lgtm

File: src/current/v24.2/change-data-capture-overview.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ For example, you might want to:
 - Export a snaphot of tables to backfill new applications.
 - Send updates to data stores for machine learning models.
 
-The main feature of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched rows".
+The main feature of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched tables".
```
Comment (rohan-joshi) [inline src/current/v24.2/change-data-capture-overview.md:18]: lgtm, thank you!

File: src/current/v24.2/changefeed-sinks.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ CockroachDB supports the following sinks:
 
 {% include {{ page.version.version }}/cdc/sink-list.md %}
 
+For a guide to using changefeeds with [Amazon Simple Notification Service (Amazon SNS)](https://docs.aws.amazon.com/sns/latest/dg/welcome.html), refer to [this blog](https://dev.to/cockroachlabs/from-cockroachdb-to-aws-sns-via-cdc-and-aws-api-gateway-bna) that uses the [webhook sink](#webhook-sink) with [Amazon API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html) to publish messages to Amazon SNS.
```
Comment (rohan-joshi) [inline src/current/v24.2/changefeed-sinks.md:14]: nice!

File: src/current/v24.3/cdc-queries.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ The following table outlines functions that are useful with CDC queries:
 
 Function                  | Description
 --------------------------+----------------------
-`changefeed_creation_timestamp()` | Returns the decimal MVCC timestamp when the changefeed was created.
+`changefeed_creation_timestamp()` | Returns the decimal MVCC timestamp when the changefeed was created. Use this function to build CDC queries that restrict emitted events by time. `changefeed_creation_timestamp()` can serve a similar purpose to the [`now()` time function]({% link {{ page.version.version }}/functions-and-operators.md %}#date-and-time-functions), which is not supported with CDC queries.
```
Comment (rohan-joshi) [inline src/current/v24.3/cdc-queries.md:64]: great

**Review by rohan-joshi — APPROVED** [review]
lgtm

File: src/current/v24.2/change-data-capture-overview.md (hunk 2 above)
Comment (mdlinville) [inline src/current/v24.2/change-data-capture-overview.md:18]:
```suggestion
The main component of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched tables".
```

File: src/current/v24.3/change-data-capture-overview.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ For example, you might want to:
 - Export a snaphot of tables to backfill new applications.
 - Send updates to data stores for machine learning models.
 
-The main feature of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched rows".
+The main feature of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched tables".
```
Comment (mdlinville) [inline src/current/v24.3/change-data-capture-overview.md:18]:
```suggestion
The main component of CockroachDB CDC is the _changefeed_, which targets an allowlist of tables, or "watched tables".
```

**Review by mdlinville — APPROVED** [review]
One totally optional suggestion

**Pull request merged**